File: 프로그래머스/lv1/12977. 소수 만들기/소수 만들기.java
class Solution {
    public int solution(int[] nums) {
        int len = nums.length;
        int answer = 0, sum = 0;
        for(int i=0; i<len-2; i++){
            for(int j=i+1; j<len-1; j++){
                for(int k=j+1; k<len; k++){
                    if(nums[i]%2!=0 || nums[k]%2!=0 || nums[j]%2!=0){
                        sum = nums[i]+nums[j]+nums[k];
                        for(int l=2; l<=sum; l++){
                            if(l==sum)
                                answer++;
                            else if(sum%l==0)
                                break;
                        }
                    }
                }
            }
        }

        return answer;
    }
}

File: 프로그래머스/1/12977. 소수 만들기/소수 만들기.js
function solution(nums) {
    var answer = 0;
    
    for(let i=0; i<nums.length-2; i++) {
        for(let j=i+1; j<nums.length-1; j++) {
            for(let k=j+1; k<nums.length; k++) {
                let num = nums[i]+nums[j]+nums[k];
                answer++;
                for(let n=num-1; n>1; n--) {
                    if(num%n === 0) {
                        answer--;
                        break;
                    }
                }
            }
        }
    }
    
    return answer;
}